frontend/test/__support__/server-mocks/metric.ts

```diff
@@ -6,6 +6,7 @@ export function setupMetricEndpoint(metric: Metric) {
 }
 
 export function setupMetricsEndpoints(metrics: Metric[]) {
-  fetchMock.get(`path:/api/metric`, metrics);
+  fetchMock.post("path:/api/metric", () => ({}));
```

Comment:
As long as it's an object, `normalizr` will take care of the rest.

Reply:
There is a pattern for handling post calls where we create mock values. Could you do the same here?